src/main/osd/osd_elements.c

```diff
@@ -156,6 +157,40 @@ static uint32_t blinkBits[(OSD_ITEM_COUNT + 31) / 32];
 #define IS_BLINK(item) (blinkBits[(item) / 32] & (1 << ((item) % 32)))
 #define BLINK(item) (IS_BLINK(item) && blinkState)
 
+#if defined(USE_ESC_SENSOR) || defined(USE_RPM_FILTER)
+
+typedef int (*getEscRpmOrFreqFnPtr)(int i);
+
+int getEscRpm(int i)
```

Comment:
Please make `static`

Reply:
it is fixed, good idea to make it local only.